### PR TITLE
Fix big merge review

### PR DESCRIPTION
**Pull request by ahlem-jarrar**
Fixes : 
T-And-1048 - Quick exit should lock the app (regardless of lock timeout configuration) 
T-And-1049 - Lock timeout = inmediately should close the app when minimized
T-And-1050 - Quick delete settings not working
T-And-1073 - Crash when going to submitted reports in Uwazi tab
T-And-1075 - crash when pressing Submit later or Outbox tabs in Uwazi 
T-And-1041 - taking photos or videos on verification mode take you back to homescreen
T-And-1069 - View icon on submitted (3 dots) should have an eye, not a pencil
T-And-1002 - Editing a report should have different actions
T-And-1072 - Uwazi logo on homescreen looks really ugly
T-And-1070 - Show title “Tella files” on the homescreen Design /App
T-And-975 - Change links on the About page 
T-And-1064 - Remove the Verification iforrmation setting on the Server setting screen
T-And-997 - Review required fields for submission,  submit later and save as draft

### CODE REVIEW

File: mobile/src/main/java/rs/readahead/washington/mobile/data/sharedpref/Preferences.java [hunk 1]
```diff
@@ -96,6 +98,14 @@ public static void setDeleteServerSettingsActive(boolean value) {
         setBoolean(SharedPrefs.DELETE_SERVER_SETTINGS, value);
     }
 
+    public boolean isEraseGalleryActive() {
```
Comment (djordjemarusic) [inline mobile/src/main/java/rs/readahead/washington/mobile/data/sharedpref/Preferences.java:101]: Why was this added?

File: mobile/src/main/java/rs/readahead/washington/mobile/data/sharedpref/Preferences.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package rs.readahead.washington.mobile.data.sharedpref;
 
+import static rs.readahead.washington.mobile.data.sharedpref.SharedPrefs.ERASE_GALLERY;
```
Comment (djordjemarusic) [inline mobile/src/main/java/rs/readahead/washington/mobile/data/sharedpref/Preferences.java:3]: We don't need to import key like this

**Review by wafalakhal — APPROVED** [review]
